src/Dapr.Actors/Communication/ActorCommunicationException.cs

```diff
@@ -0,0 +1,134 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace Dapr.Actors.Communication
+{
+    using System;
+    using System.Globalization;
+    using System.IO;
+    using System.Runtime.Serialization;
+    using System.Text;
+    using System.Xml;
+    using Dapr.Actors.Resources;
+    using Microsoft.Extensions.Logging;
+
+    /// <summary>
+    /// Provides an information about an exception from the service. This exception is thrown when the actual
+    /// exception from the service cannot be serialized for transferring to client.
+    /// </summary>
+    public class ActorCommunicationException : Exception
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ActorCommunicationException"/> class.
+        /// </summary>
+        public ActorCommunicationException()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ActorCommunicationException" /> class with appropriate message.
+        /// </summary>
+        /// <param name="actualExceptionType">the ActualExceptionType of exception thrown.</param>
```

Comment:
```suggestion
        /// <param name="actualExceptionType">The fully-qualified name of the exception type throw by the actor.</param>
```

Reply:
I will make this change with the other comments.